src/pykiso/cli.py

```diff
@@ -89,13 +89,19 @@ def check_file_extension(click_context: click.Context, param: click.Parameter, p
     return paths
 
 
-def active_threads() -> list[str]:
+def active_threads() -> list[list[str]]:
```

Comment:
Is it not ?
```suggestion
def active_threads() -> list[tuple[str, str]]:
```

Reply:
changed the return value and typing to list[tuple[str, str]]